extensions/wikia/EditPageLayout/js/plugins/TemplateClassificationEditorPlugin.js

```diff
@@ -12,14 +12,14 @@
 		WE.plugins.templateclassificationeditorplugin = $.createClass(WE.plugin, {
 
 			init: function () {
-				this.editor.on('save', this.proxy(this.forceType));
+						this.editor.on('save', this.proxy(this.forceType));
 				require(['TemplateClassificationModalForce'], function forceTemplateClassificationModal(tcForce) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this function anonymous by removing its name: 'function() {...}'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANamedFunctionExpression)

[project-id]: wikia:app